graylog2-web-interface/src/routing/AppFacade.jsx

```diff
@@ -10,13 +10,12 @@ const SessionStore = StoreProvider.getStore('Session');
 const ServerAvailabilityStore = StoreProvider.getStore('ServerAvailability');
 const CurrentUserStore = StoreProvider.getStore('CurrentUser');
 
-import 'javascripts/shims/styles/shim.css';
-import 'stylesheets/bootstrap.min.css';
+import 'bootstrap/dist/css/bootstrap.css';
 import 'stylesheets/font-awesome.min.css';
 import 'stylesheets/newfonts.less';
 import 'stylesheets/bootstrap-submenus.less';
-import 'stylesheets/toastr.min.css';
-import 'stylesheets/rickshaw.min.css';
+import 'toastr/toastr.less';
+import 'rickshaw/rickshaw.css';
```

Comment:
Shouldn't this be [`rickshaw/rickshaw.min.css`](https://github.com/shutterstock/rickshaw/blob/v1.5.1/rickshaw.min.css)?